src/WebService/Client.php

```diff
@@ -98,6 +103,9 @@ public function __construct(
         if (isset($options['host'])) {
             $this->host = $options['host'];
         }
+        if (isset($options['protocol'])) {
```

Comment:
This is part of the public API and should be documented. Perhaps we could make it a boolean instead, e.g., `useHttps`.

If we do leave it as a string, it would probably be better called `scheme` and we should leave the `://` off of it.